### PR TITLE
Release 3.1.1

### PR DESCRIPTION
**Pull request by FinalAngel**
- use default naming space ``fa-`` for fontawesome

### CODE REVIEW

File: private/sass/layout/_iconography.scss [hunk 1]
```diff
@@ -3,19 +3,18 @@
 
 // the full set of font awesome is included as default, this allows for a wider choice rather than the Glyphicons
 // for a custom set generate the font using fontastic.me
-
 @font-face {
-    font-family: "icons";
+    font-family: "FontAwesome";
+    src: url("../fonts/fontawesome/fontawesome.eot?v=4.3.0");
+    src: url("../fonts/fontawesome/fontawesome.eot?#iefix&v=4.3.0") format("embedded-opentype"),
+         url("../fonts/fontawesome/fontawesome.woff2?v=4.3.0") format("woff2"),
```
Comment (vxsx) [inline private/sass/layout/_iconography.scss]: me gusta

File: private/sass/layout/_iconography.scss [hunk 2]
```diff
@@ -24,1633 +23,1633 @@
 // INFO: use unicode characters for accessibility reasons and use aria-hidden="true" for decorative icons
 // DOCS: http://filamentgroup.com/lab/bulletproof_icon_fonts.html
 .fa-glass:before {
-  content: "\f000";
+    content: "\f000";
```
Comment (intellectronica) [inline private/sass/layout/_iconography.scss]: General comment: when you include many formatting changes in your PR, mixed with more substantial changes, you're making it harder to review and more likely that the reviewer will miss important bits. If it all possible, it's best to do the reformatting in a separate branch.

Comment (FinalAngel) [inline private/sass/layout/_iconography.scss]: I agree, yet it comes in hand as we have to use the correct formatting. In a global range, reformatting code should be added within a separate PR.
